packages/@react-spectrum/list/src/listview.css

```diff
@@ -23,15 +25,44 @@
   padding: 0;
   background-color: var(--spectrum-table-background-color, var(--spectrum-global-color-gray-50));
   outline: 0;
+  user-select: none;
 
-  [role="row"] {
+  .react-spectrum-ListView-row {
     outline: none;
   }
+
+  /* give first and last items border-radius to match listview container */
+  &:not(.react-spectrum-ListView--quiet) .react-spectrum-ListViewItem--firstItem {
+    border-start-start-radius: var(--spectrum-listview-item-start-end-border-radius);
+    border-start-end-radius: var(--spectrum-listview-item-start-end-border-radius);
+  }
+  &:not(.react-spectrum-ListView--quiet):not(.react-spectrum-ListView--loadingMore) .react-spectrum-ListViewItem--lastItem {
+    border-end-start-radius: var(--spectrum-listview-item-start-end-border-radius);
+    border-end-end-radius: var(--spectrum-listview-item-start-end-border-radius);
+  }
+}
+
+.react-spectrum-ListView-row {
+  cursor: default;
+
+  &:focus-ring {
+    &:before {
+      content: '';
+      position: absolute;
+      top: 0;
+      bottom: 0;
+      left: 0px;
```

Comment:
I don't see a `right: 0;` on this one (like above) so I'm wondering if this should be `inset-inline-start` instead? also 0 instead of 0px?

Reply:
good catch, ListView RTL has the focus indicator on the wrong side